src/nvim/api/ui_events.in.h

```diff
@@ -58,6 +58,8 @@ void set_title(String title)
     FUNC_API_SINCE(3);
 void set_icon(String icon)
     FUNC_API_SINCE(3);
+void highlights_changed(Array highlights_changed)
```

Comment:
The naming precedent is "set", we don't have "changed" (though we do have "change"... oh dear) and the distinction isn't meaningful in this case. We already have `highlight_set`, which is a UI change, whereas this new event is a property change. 

Kind of a mess. Naming this `highlight_info_set` at least is analogous to `mode_info_set`. We can use "info" as the pattern for "property changes".